lib/src/Pos/Launcher/Scenario.hs

```diff
@@ -101,7 +105,18 @@ runNode' NodeResources {..} workers' plugins' = ActionSpec $ \diffusion -> ntpCh
             sformat ("Worker/plugin with logger name "%shown%
                     " failed with exception: "%shown)
             loggerName e
-    ntpCheck = withNtpCheck $ ntpSettings onNtpStatusLogWarning
+
+    ntpCheck False action = action
+    ntpCheck True action = do
+        tvar <- newTVarIO Nothing
+        let settings = ntpSettings (ncNtpConfig nrContext) tvar
+        withAsync
+            (do
+                st <- atomically $ readTVar tvar >>= \case
+                    Nothing -> retry
+                    Just st -> return st
+                onNtpStatusLogWarning st)
```

Comment:
This is here to log in case NTP takes too long?

Reply:
No, rather this will just log the `NtpStatus`, i.e. weather we are within the threshold configured in `NtpClientSettings` (the `ntpTimeDifferenceWarnThreshold` which is copied from `NtpConfiguration`). This run asynchronously, as it was before.

Reply:
Oh this was being done before this patch?

Reply:
Yes